File: app/src/main/java/life/plank/juna/zone/data/network/model/firebaseModel/BoardNotification.java
package life.plank.juna.zone.data.network.model.firebaseModel;


import lombok.Data;

@Data
public class BoardNotification {

    private String contentType;
    private String thumbnailImageUrl;
    private Integer thumbnailWidth;
    private Integer thumbnailHeight;
    private String imageUrl;
    private String action;
    private String actor;
    private String title;
    private long foreignId;

    //TODO: Change this to camelcase once done on backend
    private String UserId;
    private String InviteeUserId;
    private String InvitationLink;
    private String InviterName;
    private String BoardId;
}

File: app/src/main/java/life/plank/juna/zone/data/model/firebase/BoardNotification.kt
package life.plank.juna.zone.data.model.firebase

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class BoardNotification(
        var contentType: String = "",
        var thumbnailImageUrl: String = "",
        var thumbnailWidth: Int = 0,
        var thumbnailHeight: Int = 0,
        var imageUrl: String = "",
        var action: String = "",
        var actor: String = "",
        var title: String = "",
        var foreignId: Long = 0,

        var userId: String = "",
        var inviteeUserId: String = "",
        var invitationLink: String = "",
        var inviterName: String = "",
        var boardId: String = ""
) : Parcelable